### ISSUE TITLE
Sass Lint doesn't apply subrules beyond the first subrule.

### ISSUE THREAD

**Issue by M1ras** [sasstools/sass-lint#1177]
Sass Lint: 1.4.0 to 1.12.1

For rules like [`class-name-format`](https://github.com/sasstools/sass-lint/blob/develop/docs/rules/class-name-format.md) and [`clean-import-paths`](https://github.com/sasstools/sass-lint/blob/develop/docs/rules/clean-import-paths.md) that have more than one subrule excluding severity, Sass Lint won't apply subrules beyond the first subrule.

# Example 1
**SassLint-Config.yaml**
```yaml
rules:
  class-name-format:
    - 1
    - allow-leading-underscore: false
    - convention: ^[a-zA-Z0-9]+(_[a-zA-Z0-9]+)*$
    - convention-explanation: "Omae wa mou shindeiru"
```
Sass Lint will apply
  - `- 1`
  - `- allow-leading-underscore: false `

Sass Lint won't apply
  - `- convention: ^[a-zA-Z0-9]+(_[a-zA-Z0-9]+)*$`
  - `- convention-explanation: "Omae wa mou shindeiru"`

# Example 2
I've created a project that demonstrates the issue as well as a bug fix.
Project: [https://github.com/M1ras/SassLint-Issues/tree/rules](https://github.com/M1ras/SassLint-Issues/tree/rules)
Bug Fix: https://github.com/M1ras/sass-lint/tree/rules

## Project Structure
  - `config/sass-lint.yaml` contains `class-name-format` and `clean-import-paths` rules.
  - `src/class-name-format.scss` contains code that should violate `class-name-format` rules but doesn't.
  - `src/clean-import-paths.scss` contains code that should violate `clean-import-paths` rules but doesn't.
  - `scripts/build.sh` tests violations for Sass Lint 1.4.0 to 1.12.1 as well as a bug fix. Logs are stored in `build/` as `build/sass-lint@SASS_LINT_VERSION` with the bug fix as `build/sass-lint@OmaewWaMouShindeiru`.

## Bug Fix
The bug fix follows contribution guidelines, passes all linting and tests, and is ready for a pull request.

**Comment by M1ras** [comment]
#1175

**Comment by DanPurdy** [comment]
Please see the last comment of #1175
